cdap-notifications/src/test/java/co/cask/cdap/notifications/NotificationTest.java

```diff
@@ -169,6 +167,38 @@ public void received(String notification, NotificationContext notificationContex
   }
 
   @Test
+  public void testCreateGetAndListFeeds() throws Exception {
+    // no feeds at the beginning
+    Assert.assertEquals(0, feedManager.listFeeds(namespace).size());
+    // create feed 1
+    feedManager.createFeed(FEED1);
+    // check get and list feed
+    Assert.assertEquals(FEED1, feedManager.getFeed(FEED1));
+    Assert.assertEquals(ImmutableList.of(FEED1), feedManager.listFeeds(namespace));
+
+    // create feed 2
+    feedManager.createFeed(FEED2);
+    // check get and list feed
+    Assert.assertEquals(FEED2, feedManager.getFeed(FEED2));
+    Assert.assertTrue(checkFeedList(ImmutableSet.of(FEED1, FEED2), feedManager.listFeeds(namespace)));
+
+    // clear the feeds
+    feedManager.deleteFeed(FEED1);
+    feedManager.deleteFeed(FEED2);
+    namespaceClient.delete(namespace);
+    Assert.assertEquals(0, feedManager.listFeeds(namespace).size());
+  }
+
+  private boolean checkFeedList(ImmutableSet<Id.NotificationFeed> expected, List<Id.NotificationFeed> actual) {
```

Comment:
I think you should also assert that their size is equal.
Else the following assertion will pass, when it shouldn't:
`checkFeedList(  Set of size 5, List of size 0 );`